src/legacy/core_plugins/console/np_ready/public/application/components/console_menu.tsx

```diff
@@ -91,6 +103,41 @@ export class ConsoleMenu extends Component<Props, State> {
     this.props.autoIndent(event);
   };
 
+  getCopyToast = () => {
+    const copyToast = [
+      {
+        title: i18n.translate('console.copyRequestAsCurl', {
```

Comment:
Could we please change this to `console.consoleMenu.copyRequestAsCurlMessage`? It helps if these ids give a description of what the text being translated does on the page.

Reply:
Ok, done.